CONTRIBUTING.md

```diff
@@ -65,6 +65,11 @@ is difficult to describe as one unit of work, please consider splitting it into
     Issues are assigned [labels](#issue-labels) which explain how they relate to the overall project's
     goals and immediate next steps.
 
+    Sometimes issues may not produce action items, and conversation will stall after a few months. 
+    When this happens, they may be marked stale by [stale-bot][link_stale-bot], 
+    and will be closed after a week unless there is more discussion. 
```

Comment:
Can we add a note here about how folks can jump in and keep the issues from closing if they think the discussion isn't resolved ?

Reply:
Is cfe50d8 what you would like?